main.js

```diff
@@ -0,0 +1,6 @@
+document.querySelector("li").remove();
+document.querySelector("section.featured span").remove();
+let artArr = document.querySelectorAll("section.posts article");
+artArr[artArr.lenght-1].remove();
+let newArr = document.querySelector("section.psots article h3");
+newArr.forEach(el=>el.remove());
```

Comment:
```suggestion
let artArr = document.querySelectorAll("section.posts article");
artArr[artArr.length-1].remove();
let newArr = document.querySelectorAll("section.posts article h3"); 
newArr.forEach(el=>el.remove());
```
You have 2 typos (`lenght` on line 4 and `psots` on line 5) and one error (`querySelector` should have been `querySelectorAll` on line 3).

These small mistakes are causing a few type errors that are keeping your tests from running at all. Other than that, you were very much on the right track.